option py_generic_services = true;

message GetTokenRequest {
    required int32 number = 1;
}

message GetTokenResponse {
    required int64 token = 1;
}

service GetTokenService {
    rpc GetToken(GetTokenRequest) returns(GetTokenResponse);
}
